README.md

```diff
@@ -154,6 +154,30 @@ jobs:
       - run: java -cp java HelloWorldApp
 ```
 
+### Install multiple JDKs
+```yaml
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
Is the checkout step required for installing the JDK?

Reply:
No it is not required, but was kept for sake of uniform with the other use cases.  I think you are right it should be removed from the snippet